browser/src/control/IFrameDialog.js

```diff
@@ -79,6 +106,32 @@ L.IFrameDialog = L.Class.extend({
 				window.postMessage('{"MessageId":"' + msg + '"}');
 			}
 		}, 1000);
+
+		if (this.options.stylesheets) {
+			this.addStyleSheets(this.options.stylesheets);
+		}
+	},
+
+	addStyleSheet: function (href) {
```

Comment:
not sure this would be the best way to append css.

Reply:
`browser/css/settings.css` should only be added when `adminIntegratorSettings.html` is used for the iframe dialog (not when used in the integrator's settings page).